rnaseq/workflow.cwl

```diff
@@ -36,14 +36,27 @@ inputs:
         type: int
     trimming_min_readlength:
         type: int
-    kallisto_index: 
+    kallisto_index:
        type: File
     gene_transcript_lookup_table:
        type: File
     firststrand:
        type: boolean?
     secondstrand:
        type: boolean?
+    refFlat:
+        type: File
+    ribosomal_intervals:
+        type: File
+    strand:
+        type: string
+    metrics_output_file:
+        type: string
+    chart_output_file:
+        type: string
+    bam:
```

Comment:
bam shouldn't be an input here, right? QC steps should be taking the aligned_bam produced by the hisat step